app/controllers/casa_cases_controller.rb

```diff
@@ -63,7 +63,7 @@ def create
   def update
     authorize @casa_case
     original_attributes = @casa_case.full_attributes_hash
-
+    @casa_case.validate_contact_type = true unless current_role == "Volunteer"
```

Comment:
the rest of the time it's false? this feels odd but if it works we will go with it :)
maybe we could make a declarative method name about this

Reply:
yeah it's a good idea